scripts/deblur_image.py

```diff
@@ -1,35 +1,38 @@
 import numpy as np
 from PIL import Image
 import click
+import os
+
 
 from deblurgan.model import generator_model
 from deblurgan.utils import load_image, deprocess_image, preprocess_image
 
 
-def deblur(image_path):
-    data = {
-        'A_paths': [image_path],
-        'A': np.array([preprocess_image(load_image(image_path))])
-    }
-    x_test = data['A']
-    g = generator_model()
-    g.load_weights('generator.h5')
-    generated_images = g.predict(x=x_test)
-    generated = np.array([deprocess_image(img) for img in generated_images])
-    x_test = deprocess_image(x_test)
-
-    for i in range(generated_images.shape[0]):
-        x = x_test[i, :, :, :]
-        img = generated[i, :, :, :]
-        output = np.concatenate((x, img), axis=1)
-        im = Image.fromarray(output.astype(np.uint8))
-        im.save('deblur'+image_path)
+def deblur(weight_path, input_dir, output_dir):
+	input_dir = input_dir if input_dir[-1] == '/' else input_dir + '/'
+	output_dir = output_dir if output_dir[-1] == '/' else output_dir + '/'
+	g = generator_model()
+	g.load_weights(weight_path)
+	for image_name in os.listdir(input_dir):
+	    image = np.array([preprocess_image(load_image(input_dir + image_name))])
+	    x_test = image
+	    generated_images = g.predict(x=x_test)
+	    generated = np.array([deprocess_image(img) for img in generated_images])
+	    x_test = deprocess_image(x_test)
+	    for i in range(generated_images.shape[0]):
+	        x = x_test[i, :, :, :]
+	        img = generated[i, :, :, :]
+	        output = np.concatenate((x, img), axis=1)
+	        im = Image.fromarray(output.astype(np.uint8))
+	        im.save(output_dir + image_name)
```

Comment:
Same here, you can use `os.path.join(output_dir, image_name)`

Reply:
Ok. I fixed it already.